mysqld_exporter.go

```diff
@@ -195,7 +196,7 @@ func newHandler(metrics collector.Metrics, scrapers []collector.Scraper, logger
 				r = r.WithContext(ctx)
 			}
 		}
-		level.Debug(logger).Log("msg", "collect[] params", "params", params)
+		level.Debug(logger).Log("msg", "collect[] params", "params", strings.Join(params, "+"))
```

Comment:
I think a comma would be better.

```suggestion
		level.Debug(logger).Log("msg", "collect[] params", "params", strings.Join(params, ","))
```

Reply:
Thanks. I got it!